src/client/retry.rs

```diff
@@ -622,7 +622,7 @@ mod tests {
         assert_eq!(r.status(), StatusCode::OK);
 
         // Handles redirect loop
-        for _ in 0..10 {
+        for _ in 0..11 {
```

Comment:
I believe this change is needed due to https://github.com/seanmonstar/reqwest/pull/2664

without this change the test fails because the redirect loop fails to trigger the limit